### PR TITLE
limit plugin to specific stages

### LINKED ISSUE #55: Add ability to include/exclude layer for specific stages
A big part of serverless is the ability to be able to deploy your same application stack multiple times across multiple accounts by specifying different stages.

It would be great if it was possible to only enable newrelic for some of these stages. E.g.

```
custom:
  newrelic:
    stages:
      - prod    # where provider.stage = prod
````

### PR DESCRIPTION
**Pull request by karopolopoulos**
Fixes #55

### CODE REVIEW

**Comment by karopolopoulos** [comment]
One issue I have with the way I did things was the fact that each hook needs to now run the `shouldSkipPlugin()` function.
Looking for a way instead to just unbind all the hooks before loading.

May move the running of this function to the constructor

**Comment by karopolopoulos** [comment]
Also, I got the versioning wrong

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-newrelic-lambda-layers",
-  "version": "0.1.18",
+  "version": "0.2.18",
```
Comment (kolanos) [inline package.json:3]: This change doesn't break API backwards compatibility. So could just be `0.1.19`.

Comment (mrickard) [inline package.json:3]: I'd think this should be `0.2.0`, shouldn't it?

**Comment by mrickard** [comment]
LGTM; I'd be happy to approve when the versioning's resolved.

**Comment by kolanos** [comment]
Going to merge this and revise version in subsequent commit.

**Pull request merged**